### ISSUE TITLE
More Signal Converter Errors

### ISSUE THREAD

**Issue by Doc-1** [CreativeMD/LittleTiles#585]
It's not the same error however, I and another person on discord, Hiru, still crashes when a signal is sent to it but unfortunately we don't have the same crashes. I even get two different crashes during my tests. We both have the latest versions LT pre318, and CC v1.10.44. Lastly, I am able to use the Signal Converter in the dev environment just fine. Only when it is compiled is there an issue. I've provided the crash that Hiru submitted on discord as well to keep it all in one place.
Mine, Abstract World
crash-2021-01-22_11.39.23-server.txt
Mine, No ClassDef Error.
crash-2021-01-22_11.29.38-server.txt
Hiru, Index out of bounds
crash-2021-01-22_13.05.55-server.txt

**Comment by CreativeMD** [comment]
Some of these might be fixed in the newest version. So would be very nice if you can test it again.

**Comment by Doc-1** [comment]
Yup, it fixed my issue that I was having (This time, I didn't run it in the dev environment) The only thing I found was when I first attempted to join the world that previously crashed due to the signal converter. It posted a looping error to the log then crashed. I had to join the world a second time for it to work. If you want me to post this in a new issue post, let me know.

Just the looped part
[Looping Error from log file.txt](https://github.com/CreativeMD/LittleTiles/files/5860105/Looping.Error.from.log.file.txt)
The entire log
[latest.log](https://github.com/CreativeMD/LittleTiles/files/5860107/latest.log)
The crash report.
[crash-2021-01-23_06.24.12-server.txt](https://github.com/CreativeMD/LittleTiles/files/5860109/crash-2021-01-23_06.24.12-server.txt)
